spec.bs

```diff
@@ -1234,12 +1244,11 @@ Finally, at the end of the same nested scope, add the following step:
 </div>
 
 The {{WindowSharedStorage}}'s {{WindowSharedStorage/selectURL()}} method steps
-are modified in three ways. First, add the following steps just after step 5
-("If {{Worklet/addModule()}} has not yet been called, ..."), renumbering later
-steps:
+are modified in three ways. First, add the following steps just after step 6 ("If context is null
```

Comment:
nit: `"If <var ignore>context</var> is null ..."`

Reply:
Added `<var ignore>`.